tests/functional_tests/test_general_formatting.py

```diff
@@ -97,3 +97,21 @@ def test_formatting(p: str) -> None:
 
     second_pass = format_string(actual, mode)
     check_formatting(expected, second_pass, ctx=f"2nd-{p}")
+
+
+@pytest.mark.parametrize(
+    "p",
+    [
```

Comment:
I think you missed a couple.
```suggestion
    [
        "preformatted/008_markdown_file.md",
        "preformatted/009_markdown_fmt_off.md",
```